cmsl1t/recalc/met.py

```diff
@@ -2,8 +2,27 @@
 import numpy as np
 
 
+class met(object):
+    def __init__(self, metx, mety):
+        self.metx = metx
+        self.mety = mety
```

Comment:
Probably better to name them `x` and `y` so you have `met.x` instead of `met.metx`.